docs/citycloud-snippets.md

```diff
@@ -0,0 +1,130 @@
+# Citycloud snippets
+
+To set up a cluster on citycloud these snippets can be used
+
+Start by setting up some environments for this setup
+
+```bash
+SERVICE_CLUSTER="my-new-sc-cluster"
+WORKLOAD_CLUSTERS=( "my-new-wc-cluster" )
+PUB_SSH_KEY_FILE="${HOME}/.ssh/id_rsa.pub"
+```
+
+> Tip:
+> Save these into a file `my-new-cluster.env`.
+> The next time you want to use these snippets you can just run `source my-new-cluster.env`
+
+Then you'll need to set up some openstack specific variables for the infrastructure
+
+```bash
+OS_IMAGE_NAME="Ubuntu 20.04 Focal Fossa 20200423" # Image name to base VMs on
```

Comment:
Yes, I should add a short mentioning of how to authenticate. Maybe link to some documentation.

The reason is mainly because I wanted to, once again,  align with the aws / exoscale guide